dijet/production/dijet_balance.py

```diff
@@ -4,43 +4,98 @@
 Selectors to set ak columns for dijet properties
 """
 
+# TODO: Not all columns needed are present yet for pt balance method
+#       - Include probe and reference jet from jet assignment
+#       - is FE or SM Method
+
 from columnflow.util import maybe_import
 from columnflow.columnar_util import set_ak_column, EMPTY_FLOAT  # , Route
 from columnflow.selection import SelectionResult
 from columnflow.production import Producer, producer
+from columnflow.calibration.util import ak_random
+
+from dijet.constants import eta
 
 np = maybe_import("numpy")
 ak = maybe_import("awkward")
 
 @producer(
-    uses={"Jet.pt"},
-    produces={"dijets.pt_avg","dijets.asymmetry","dijets.alpha"}
+    uses={
+        "Jet.pt", "Jet.eta",
+        },
+    produces={
+        "probe_jet.pt", "probe_jet.eta",
+        "reference_jet.pt", "reference_jet.eta",
+        "dijets.pt_avg","dijets.asymmetry","dijets.alpha"
+        }
```

Comment:
All output columns should appear here, otherwise they won't be written to the output.
```suggestion
    produces={
        "probe_jet.pt", "probe_jet.eta",
        "reference_jet.pt", "reference_jet.eta",
        "dijets.pt_avg", "dijets.asymmetry", "dijets.alpha", "dijets.is_sm", "dijets.is_fe",
    },
```